fenix/app/src/main/AndroidManifest.xml

```diff
@@ -333,7 +333,7 @@
                 android:exported="false" />
 
         <service
-            android:name=".gleanplumb.NotificationDismissedService"
+            android:name=".messaging.NotificationDismissedService"
```

Comment:
Does this `AndroidManifest.xml` fragment need to go to `service-nimbus`?
Answer: No. 

It might do if MessageNotificationWorker needs to go to an Android Component (though likely not the `service-nimbus` one).

Reply:
I filed a bug to track that [here](https://bugzilla.mozilla.org/show_bug.cgi?id=1823736)